modules/gcloud/src/test/java/org/testcontainers/containers/DatastoreEmulatorContainerTest.java

```diff
@@ -0,0 +1,41 @@
+package org.testcontainers.containers;
+
+import com.google.cloud.NoCredentials;
+import com.google.cloud.ServiceOptions;
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.Key;
+import org.junit.Rule;
+import org.junit.Test;
+import org.testcontainers.utility.DockerImageName;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class DatastoreEmulatorContainerTest {
+
+    @Rule
+    // creatingDatastoreEmulatorContainer {
+    public DatastoreEmulatorContainer emulator = new DatastoreEmulatorContainer(DockerImageName.parse("gcr.io/google.com/cloudsdktool/cloud-sdk:313.0.0"));
```

Comment:
When rendered, this has to be scrolled quite a bit. Let's perhaps wrap it?
```suggestion
    public DatastoreEmulatorContainer emulator = new DatastoreEmulatorContainer(
        DockerImageName.parse("gcr.io/google.com/cloudsdktool/cloud-sdk:313.0.0")
    );
```